libsrc/apple2/cputc.s

```diff
@@ -5,13 +5,15 @@
 ; void __fastcall__ cputc (char c);
 ;
 
-        .ifdef  __APPLE2ENH__
-        .constructor    initconio
-        .endif
+        ; Call constructor *after* the 80-columns card detection
+        ; in videomode.s
+        .constructor    initconio, 7
```

Comment:
As far as I know, 7 is the default priority. It's rather normal that default priority constructors implicitly rely on higher priority contructors having run first. There's no need/desire to exolictly mention this and/or explicitly set the default priority. If there's a need/desire to clarify things then they should be mentioned at the higher priority constructor explaining why the explicitly given priority needs to be higher there.